overview/introduction.mdx

```diff
@@ -33,11 +33,37 @@ Novu additionally empowers your team with insights into logs and analytics, faci
 
 Novu achieves notification delivery across both in-app and external channels. In the case of in-app channels, we offer pre-built components for promptly embedding a fully functional notification feed into your application within minutes. As for external channels, Novu seamlessly integrates with your existing providers – whether email, push, SMS services, or platforms like Slack and Apple Push Notification Service.
 
-## Product vs. Promotional Notifications
+## **Product vs. Promotional Notifications**
 
-At Novu, our focus is squarely on product notifications (often transactional ones). These notifications are typically triggered by subsciber (user) events within your product or time-sensitive queries, requiring notification of one or many other subscribers.
+**Product Notifications**
+
+- Triggered by subscriber events within your product or time-sensitive queries
+- Notify one or many subscribers
+- Typically transactional, such as order updates, password resets, or account verifications
+- Can also be used to provide product updates, such as new features or bug fixes
+
+**Promotional Notifications**
+
+- Disseminated to specific user segments or prospects
+- Centered on promoting products, services, or events
+- Can be used to generate leads, increase sales, or drive engagement
+
+**Key Differences**
+
+- Product notifications are triggered by user activity, while promotional notifications are sent at the discretion of the marketer
+- Product notifications are typically more transactional in nature, while promotional notifications are more marketing-oriented
+- Product notifications are often sent to all subscribers, while promotional notifications can be targeted to specific audiences
+
+<Tip>To ensure our alignment, please familiarize yourself with the terms that you are likely to come across while perusing this documentation or utilizing Novu.</Tip>
+
+| Term           | Definition                                                                                          |
+|----------------|----------------------------------------------------------------------------------------------------|
+| Feed           | A list of all In-App messages for a single subscriber                                              |
+| Message        | A content sent to a single subscriber over a single channel                                      |
+| Workflow       | A sequence of steps involved in sending messages, digesting multiple events in one, filtering & delaying |
+| Notification   | A workflow run in the context of a single subscriber. A combination of messages sent to all activated channels |
+| Trigger Event  | An execution of a workflow for a single or multiple subscribers from start to end                 |
```

Comment:
Please can you add full stop at the end of each sentences? @iampearceman

Reply:
Done.